bandit/runner.h

```diff
@@ -65,22 +65,22 @@ namespace bandit {
 
   inline void use_default_reporters(detail::choice_options& choices) {
     choices.reporters.add("singleline", [&](detail::controller_t& controller) {
-      controller.set_reporter(new bandit::reporter::singleline(controller.get_formatter(), controller.get_colorizer()));
+      controller.set_reporter(new bandit::reporter::singleline(controller.get_formatter(), controller.get_report_timing(), controller.get_colorizer()));
     });
     choices.reporters.add("xunit", [&](detail::controller_t& controller) {
-      controller.set_reporter(new bandit::reporter::xunit(controller.get_formatter()));
+      controller.set_reporter(new bandit::reporter::xunit(controller.get_formatter(), controller.get_report_timing()));
     });
     choices.reporters.add("info", [&](detail::controller_t& controller) {
-      controller.set_reporter(new bandit::reporter::info(controller.get_formatter(), controller.get_colorizer()));
+      controller.set_reporter(new bandit::reporter::info(controller.get_formatter(), controller.get_report_timing(), controller.get_colorizer()));
     });
     choices.reporters.add("spec", [&](detail::controller_t& controller) {
-      controller.set_reporter(new bandit::reporter::spec(controller.get_formatter(), controller.get_colorizer()));
+      controller.set_reporter(new bandit::reporter::spec(controller.get_formatter(), controller.get_report_timing(), controller.get_colorizer()));
     });
     choices.reporters.add("crash", [&](detail::controller_t& controller) {
-      controller.set_reporter(new bandit::reporter::crash(controller.get_formatter()));
+      controller.set_reporter(new bandit::reporter::crash(controller.get_formatter(), controller.get_report_timing()));
     });
     choices.reporters.add("dots", [&](detail::controller_t& controller) {
-      controller.set_reporter(new bandit::reporter::dots(controller.get_formatter(), controller.get_colorizer()));
+      controller.set_reporter(new bandit::reporter::dots(controller.get_formatter(), controller.get_report_timing(), controller.get_colorizer()));
```

Comment:
According to comment :abc:, it's only necessary to change the reporter invocations that support the report of timing.

Reply:
Fixed here: 043991d